src/window/glutin/src/lib.rs

```diff
@@ -1,3 +1,27 @@
+//! Window creation using glutin for gfx.
+//!
+//! # Examples
+//! The following code creates a `gfx::WindowExt` using glutin.
```

Comment:
hmm, it doens't make sense to duplicate the same code here and in the glutin window example. Let's make this one fully compilable and remove the latter?

Reply:
Sorry, what other glutin window example is that?

Reply:
Ah, pardon me, I was thinking about GLFW/SDL